commons/src/main/java/io/aiven/kafka/connect/common/config/AivenCommonConfig.java

```diff
@@ -55,151 +55,104 @@ public class AivenCommonConfig extends AbstractConfig {
-                                                           final OutputFieldType defaultFieldType) {
+            final OutputFieldType defaultFieldType) {
         int formatGroupCounter = 0;
 
         addFormatTypeConfig(configDef, formatGroupCounter);
 
-        configDef.define(
-            FORMAT_OUTPUT_FIELDS_CONFIG,
-            ConfigDef.Type.LIST,
-            !Objects.isNull(defaultFieldType) ? defaultFieldType.name : null,
-            new OutputFieldsValidator(),
-            ConfigDef.Importance.MEDIUM,
-            "Fields to put into output files. "
-                + "The supported values are: " + OutputField.SUPPORTED_OUTPUT_FIELDS + ".",
-            GROUP_FORMAT,
-            formatGroupCounter++,
-            ConfigDef.Width.NONE,
-            FORMAT_OUTPUT_FIELDS_CONFIG,
-            FixedSetRecommender.ofSupportedValues(OutputFieldType.names())
-        );
-
-        configDef.define(
-            FORMAT_OUTPUT_FIELDS_VALUE_ENCODING_CONFIG,
-            ConfigDef.Type.STRING,
-            OutputFieldEncodingType.BASE64.name,
-            new OutputFieldsEncodingValidator(),
-            ConfigDef.Importance.MEDIUM,
-            "The type of encoding for the value field. "
-                + "The supported values are: " + OutputFieldEncodingType.SUPPORTED_FIELD_ENCODING_TYPES + ".",
-            GROUP_FORMAT,
-            formatGroupCounter++,
-            ConfigDef.Width.NONE,
-            FORMAT_OUTPUT_FIELDS_VALUE_ENCODING_CONFIG,
-            FixedSetRecommender.ofSupportedValues(OutputFieldEncodingType.names())
-        );
-
-        configDef.define(
-            FORMAT_OUTPUT_ENVELOPE_CONFIG,
-            ConfigDef.Type.BOOLEAN,
-            true,
-            ConfigDef.Importance.MEDIUM,
-            "Whether to enable envelope for entries with single field.",
-            GROUP_FORMAT,
-            formatGroupCounter,
-            ConfigDef.Width.SHORT,
-            FORMAT_OUTPUT_ENVELOPE_CONFIG
-        );
-    }
-
-    protected static void addFormatTypeConfig(final ConfigDef configDef,
-                                              final int formatGroupCounter) {
-        final String supportedFormatTypes = FormatType.names().stream()
-            .map(f -> "'" + f + "'")
-            .collect(Collectors.joining(", "));
-        configDef.define(
-            FORMAT_OUTPUT_TYPE_CONFIG,
-            ConfigDef.Type.STRING,
-            FormatType.CSV.name,
-            new OutputTypeValidator(),
-            ConfigDef.Importance.MEDIUM,
-            "The format type of output content"
-                + "The supported values are: " + supportedFormatTypes + ".",
-            GROUP_FORMAT,
-            formatGroupCounter,
-            ConfigDef.Width.NONE,
-            FORMAT_OUTPUT_TYPE_CONFIG,
-            FixedSetRecommender.ofSupportedValues(FormatType.names())
-        );
+        configDef.define(FORMAT_OUTPUT_FIELDS_CONFIG, ConfigDef.Type.LIST,
```

Comment:
I believe having the parameters for `configDef.define` each on new line is a lot more readable and that's what we are trying to maintain in most of the repositories, aren't we?